src/Pages/Login.js

```diff
@@ -1,5 +1,65 @@
+import {Box, Button, TextField, Typography} from '@mui/material';
+import {useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {Navigate} from 'react-router-dom';
+import {addUser} from '../redux/redux-reducers/userstore';
+
 const Login = () => {
-  return <div>LOGIN</div>;
+  const loggedIn = useSelector((state) => state.userStore.loggedIn);
+  const dispatch = useDispatch();
+  const [email, setEmail] = useState('');
+  const handleClick = () => {
+    //checking if it's an email
+    if (email.includes('@')) {
+      dispatch(addUser(email));
+      window.location.reload();
```

Comment:
Why do we do reload if we are adding the value to the store?

Reply:
I used reload so it redirects to the homepage after clicking log in

Reply:
Replaced window.location.reload() with useNavigate() so it redirects on login